mrack.spec

```diff
@@ -1,7 +1,7 @@
 # Created by pyp2rpm-3.3.5
 %global srcname mrack
 
-Name:           %{srcname}
+Name:           mrack
```

Comment:
A nit ... you could even drop the %srcname definition now?  I don't think it was necessary to move from `srcname` to just `name`, but admittedly the spec looks prettier now.

Reply:
Thanks! Yeah I forgot to remove the `srcname`